docs/docs/recipes.md

```diff
@@ -152,6 +152,34 @@ By targeting the HTML `body` element, your font will apply to most text on the p
 - More on [importing assets into files](/docs/importing-assets-into-files/]
 - [Using Typography.js for Google fonts](/docs/typography-js/)
 
+### Using Google Fonts
+
+#### Prerequisites
+
+- A [Gatsby site](/docs/quick-start)
+- The [Gatsby CLI](/docs/gatsby-cli/) installed
+- Choosing a font package from [the typefaces project](https://github.com/KyleAMathews/typefaces)
+
+#### Directions
+
+1. Run `npm install --save yourchosenfont`, replacing `yourchosenfont` with the name of the font you want to install from [the typefaces project](https://github.com/KyleAMathews/typefaces).
+2. Add `import "yourchosenfont"` to a layout template, page component, or `gatsby-browser.js`.
```

Comment:
I'm sorry I didn't notice this in my earlier review–but I I think a code block in addition to the text would help make this step more scannable. How about something like:

```jsx:title=src/components/layout.js
import "yourchosenfont"
```